openjdk8/pom.xml

```diff
@@ -81,7 +81,7 @@
 	    <configuration>
 	      <imageName>openjdk8</imageName>
```

Comment:
Actually,  it is currently openjdk8 to match the module name and directory name, in anticipation of there soon being an openjdk9

I thought the decision was to keep the version in the name when parallel versions were likely to be maintained concurrently - and thus avoid lots of branches.

I guess we could have directory/module openjdk8 builds openjdk:8-buildID rather than openjdk8:buildID ?